### PR TITLE
Add C++ solution to USACO jan20-berry-picking.mdx

### PR DESCRIPTION
**Pull request by 23danielsuh**

### CODE REVIEW

File: solutions/USACO jan20-berry-picking.mdx [hunk 1]
```diff
@@ -11,6 +11,83 @@ author: Qi Wang
...
+        //Amount calculates how many groups of "i" berries can be put into a basket
+		for(int j = 0; j < N; j++) {
+			amount += A[j] / i;
+		}
+        //If the amount is not enough for K / 2 baskets, it is not valid
+		if(amount < K / 2) {
+			continue;
+		}
+		if(amount >= K) {
+            //If there is enough "i" sections for both Bessie and Ellie, then just have (K / 2) * i for Bessie
```
Comment (frodakcin) [inline solutions/USACO jan20-berry-picking.mdx:66]:
```suggestion
            //If there are enough "i" sections for both Bessie and Ellie, then Bessie can collect (K / 2) * i berries
```

**Review by frodakcin — CHANGES_REQUESTED** [review]
The code is all tabs, but there are spaces before the comments. Can we stick to all tabs?

**Comment by 23danielsuh** [comment]
> The code is all tabs, but there are spaces before the comments. Can we stick to all tabs?

Just changed it to tabs, and revised one of the comments. Not sure what happened, I think I edited the old file in a different text editor that defaults to spaces instead of tabs.

**Comment by frodakcin** [comment]
Yea, probably. It'd normally be impossible to notice but conveniently github uses super large tabs which makes it obvious.

**Review by frodakcin — APPROVED** [review]
Looks good.

File: solutions/USACO jan20-berry-picking.mdx [hunk 2]
```diff
@@ -2,7 +2,7 @@
 id: usaco-990
 source: USACO Silver 2020 January
 title: Berry Picking
-author: Qi Wang
+author: Qi Wang {java}, Daniel Suh {C++}
```
Comment (frodakcin) [inline solutions/USACO jan20-berry-picking.mdx:5]:
```suggestion
author: Qi Wang (Java), Daniel Suh (C++)
```

**Review by frodakcin — CHANGES_REQUESTED** [review]
Actually, one more thing. I think using parentheses is more standard

**Comment by 23danielsuh** [comment]
Yeah, that makes more sense. Just committed the changes.

**Review by frodakcin — APPROVED** [review]

**Comment by 23danielsuh** [comment]
Sorry for all of the commits, but I just changed 2 of the comments to make more sense.

**Comment by frodakcin** [comment]
No worries! Feel free to keep making changes

**Comment by frodakcin** [comment]
Will merge after @HarryWangATX 's review

**Comment by HarryWangATX** [comment]
K reviewing rn

**Review by HarryWangATX — APPROVED** [review]
doesn't the official sol include C++? welp ig its commented so doesn't hurt.

**Pull request merged**